server/controllers/calendarController.ts

```diff
@@ -30,11 +30,12 @@ export default class CalendarController {
     const notifications = !this.notificationCookieService.alreadyDismissed(req, SMS_BANNER)
 
     const computeBanner = async () => {
+      const alreadyDismissedExisting = this.notificationCookieService.alreadyDismissed(req, EXISTING_USER)
       const alreadyDismissedNew = this.notificationCookieService.alreadyDismissed(req, NEW_USER)
 
       const authMfa = await this.userService.getUserMfa(token)
       if (authMfa.backupVerified || authMfa.mobileVerified) {
-        return alreadyDismissedNew ? '' : NEW_USER
+        return alreadyDismissedNew || alreadyDismissedExisting ? '' : NEW_USER
```

Comment:
want to restrict how many people see the banner - so don't show it if they've dismissed the old existing user banner (as new one contains similar contents)